idp_data/idp_data/models.py

```diff
@@ -1,3 +1,38 @@
 from django.db import models
+from datetime import date
+from ckeditor.fields import RichTextField
 
-# Create your models here.
+
+class Category(models.Model):
+    name = models.CharField(max_length=500)
+    icon = models.CharField(max_length=50, default='')
+
+    def __str__(self):
+        return self.name
+
+
+class Municipality(models.Model):
+    code = models.CharField(max_length=50)
+
+    def __str__(self):
+        return self.code
+
+
+class Event(models.Model):
+    muni = models.ForeignKey(Municipality, on_delete=models.DO_NOTHING, default=None, blank=True, null=True)
```

Comment:
Better to disallow blank and null and cascade deletes so that we don't end up with messy data

Reply:
done

Reply:
Which class do you mean? I only see Event class here ^

Reply:
done